media/pages/sk/faq/page.html

```diff
@@ -5,108 +5,199 @@ <h1>Často kladené otázky</h1>
 
 <h2 id="o-portali">O portáli</h2>
 
-<h3 id="co-je-chcemvediet-sk">Čo je chcemvediet.sk?</h3>
-<p>
-Chcemvediet.sk je portál, prostredníctvom ktorého môžete poslať žiadosť o informácie ktorejkoľvek
-inštitúcii, ktorá má povinnosť sprístupňovať informácie v zmysle zákona o slobodnom prístupe
-k informáciám č. 211/2000 Z.z. z jedného miesta, a ďalej spravovať svoje žiadosti o informácie.
-Zároveň môžete v databáze vyhľadávať medzi informáciami, ktoré už boli sprístupnené iným žiadateľom
-a sledovať štatistiky o vybavovaní žiadostí o informácie.
-</p>
+<div class="question">
+  <h3 id="co-je-chcemvediet-sk" class="chv-pointer" data-toggle="collapse"
+      data-target="#co-je-chcemvediet-sk-text">
+    <i class="faq-collapse-shown chv-icon chv-icon-lg icon-plus"></i><i class="hidden faq-collapse-hidden chv-icon chv-icon-lg icon-minus"></i><span class="question-heading-text">Čo je chcemvediet.sk?</span>
+  </h3>
+  <div id="co-je-chcemvediet-sk-text" class="collapse" data-target-shown=".faq-collapse-shown"
+       data-target-hidden=".faq-collapse-hidden" data-container=".question"
+       data-hash="co-je-chcemvediet-sk">
+    <p class="question-text">
```

Comment:
V skutocnosti je odsadenie rovnake, ale vizualne to nie je na rovnakej urovni, pretoze `<p>` ma `font-size: 14px` a `<h3>` ma `font-size: 15px`.

Reply:
IMHO toto by bolo lepsie vyriesit rovnako ako sa to riesi pri `ul`/`li`. Keby namestio `h3` bolo `li` a namiesto `div` nad nim bolo `ul`, tak staci ikonke dat classu `chv-icon-li` a ikonka sa sama automaticky zarovna dolava pred text.

Semanticky to vsak `ul`/`li` nie je a tag `h3` chceme zachovat. Nic nam vsak nebrani do `typo.sass` k `ul`/`li` stylom pridat nove classy `chv-ul` a `chv-li` a tie pouzit pre `div`/`h3`.

To by nam myslim vyriesilo aj problem s whitespacom medzi tagmi. Kedze whitespace sa pri ikonkach v `li` ignoruje.

Cca nieco taketo by sme mali:

```
<div class="question chv-ul">
  <h3 id="co-je-chcemvediet-sk" class="chv-pointer chv-li" data-toggle="collapse" ...>
    <i class="faq-collapse-shown chv-icon chv-icon-li chv-icon-lg icon-plus"></i>
    ...
  </h3>
  ...
</div>
```

Reply:
@martinmacko47 mal si namysli v `typo.sass` pridat 
```
ul, ol, .chv-ul
  margin-top: 0
  ...

li, .chv-li
  ...
```

ten whitespace sa tym nevyriesil. Pridaval ho aj medzi elementami `<i>` a `<i>` (pridal som si jeden) a aj medzi elementami `<i>` a `<span>`. 

Po pridani danych class sa nam spravil `padding-left: 32px`.  Cize ten padding moze byt, takto si to predstavujes?
![Screenshot_2021-03-24 Často kladené otázky (FAQ) Chcemvediet sk](https://user-images.githubusercontent.com/29918752/112358800-f4053400-8cd0-11eb-9278-2d06d80325b3.png)

Reply:
@viliambalaz Ano, takto. Len pozor to `li` je tam este raz vnorene. Tak aj pre to vnorene treba `.chv-li`

S tym whitespacom som mal pocit, ze sa to vyriesilo. Ved `.chv-icon-li` ma nastavene `float: left`, takze ikonky sa vysunu mimo flowu. Tym padom vsetky medzery medzi ikonkami sa zlucia do jednej, ktora bude ako prvy inline znak v ramci block elementu. A taka medzera by sa mala ignorovat, ci?

Padding by to malo spravit rovnaky ako je na ostatnych miestach, kde sa pouziva `<ul>`. [Napriklad tu](https://www.chcemvediet.sk/sk/o-projekte/):

![padding](https://user-images.githubusercontent.com/5547275/112378139-dcd14100-8ce6-11eb-9c05-4b67b8fe4c90.png)

Reply:
:ok: funguje, v `typo.sass` su este selectory, ale prvy nechceme pouzit, pretoze chceme zachovat `h3` a `chv-dense` zatial nepotrebujeme
```
li
  @include chv-baseline($chv-fontsize-li, 1, 0, 0.5)
li.chv-dense
  @include chv-baseline($chv-fontsize-li, 1, 0, 0)
```

Reply:
Ano, tieto pravidla riesia velkost pisma, tie nechceme.